src/PkgDeps.jl

```diff
@@ -173,10 +123,11 @@ end
 """
     users(pkg_name::String, pkg_registry::RegistryInstance; kwargs...)
 
-Find the users of the package named `pkg_name` which is registered in `pkg_registry`.
+Find the users of `pkg_name` which is registered in `pkg_registry_name`.
+Optionally limit search for users of `pkg_name` in specific registries by passing in kwarg `registries`.
```

Comment:
The `registries` is already documented (since docstrings for the same method get appended together), so no need to document this in two places :)
```suggestion
Find the users of `pkg_name` which is registered in `pkg_registry`.
```